narayana/tools/src/main/java/io/narayana/perf/jmh/Benchmark.java

```diff
@@ -77,6 +77,11 @@ public String toString() {
 		if (previous != null) {
 			double percentChange = regression == 0 ? 0.0 : (score / previous.score - 1) * 100.0;
 			String changeType = regression == -1 ? "regression" : regression == 0 ? "no change" : "improvement";
+			if (regression == 0) {
+                String summary="Considering the deviation of the numbers, we do not determine this performance test to have altered significantly.";
+                return String.format("%s:%s %f(deviation:%f) vrs %f(deviation:%f) (%f%%: %s)%n", benchmark, summary, score,
+                        scoreError, previous.score, previous.scoreError, percentChange, changeType);
```

Comment:
I think we should either remove the (%f%%: %s) for the non-regression case or we should consider changing
`double percentChange = regression == 0 ? 0.0 : (score / previous.score - 1) * 100.0;`
to
`double percentChange = (score / previous.score - 1) * 100.0;`

Reply:
Okay, I will remove `(%f%%: %s)`